apps/storybook/src/VisCanvasInteraction.stories.tsx

```diff
@@ -1,24 +1,52 @@
-import type { PanZoomProps, TooltipMeshProps } from '@h5web/lib';
-import { PanZoomMesh, TooltipMesh, VisCanvas } from '@h5web/lib';
+import type { TooltipMeshProps } from '@h5web/lib';
+import {
+  PanMesh,
+  TooltipMesh,
+  VisCanvas,
+  XAxisZoomMesh,
+  YAxisZoomMesh,
+  ZoomMesh,
+} from '@h5web/lib';
+import type { ModifierKey } from '@h5web/lib/src/vis/models';
 import { formatTooltipVal } from '@h5web/shared';
 import type { Meta, Story } from '@storybook/react';
 
 import VisCanvasStoriesConfig from './VisCanvas.stories';
 
-interface TemplateProps extends PanZoomProps {
+interface TemplateProps {
   tooltipValue?: string;
   guides?: TooltipMeshProps['guides'];
+  disablePan?: boolean;
+  panKey?: ModifierKey;
+  disableZoom?: boolean;
+  disableXZoom?: boolean;
+  xZoomKey?: ModifierKey;
+  disableYZoom?: boolean;
+  yZoomKey?: ModifierKey;
 }
 
 const Template: Story<TemplateProps> = (args) => {
-  const { tooltipValue, guides, ...panZoomProps } = args;
+  const {
+    tooltipValue,
+    guides,
+    disablePan,
+    panKey,
+    disableZoom,
+    disableXZoom,
+    xZoomKey,
+    disableYZoom,
+    yZoomKey,
+  } = args;
```

Comment:
Because of all the prop aliases, I feel like this template no longer documents the new meshes very well. Perhaps we should have one template per mesh? We can always store the canvas' axis configs into a shared object that we then destructure. Might also be worth splitting all the pan/zoom interactions into a separate file to stay clear of the tooltip? All of this can be left to a future PR.

Reply:
No you are right, I might as well do it in this PR to properly document each mesh.

Reply:
So I split into two files: one for the tooltip and one for the PanZoom meshes. In PanZoom, each story documents an individual mesh with controls that reflect its props and that act only on the relevant mesh. 

But all meshes are still present in each story as `Zoom` is needed for `Pan` to be used.

Tell me what you think.